src/zimit/zimit.py

```diff
@@ -143,8 +138,8 @@ def run(raw_args):
     parser.add_argument(
         "--waitUntil",
         help="Puppeteer page.goto() condition to wait for before continuing. One of "
-        f"{wait_until_options} or a comma-separated combination of those.",
-        choices=wait_until_all,
+        f'{["load", "domcontentloaded", "networkidle0", "networkidle2"]} or a '
```

Comment:
Why using an fstring to display static value? I don't get the reasoning…

Reply:
Updated, I found it less confusing ... taste ...